webapp/src/Service/ImportExportService.php

```diff
@@ -920,7 +921,8 @@ protected function importTeamData(array $teamData, ?string &$message, ?array &$s
                     $teamAffiliation = new TeamAffiliation();
                     $teamAffiliation
                         ->setExternalid($teamItem['team_affiliation']['externalid'])
-                        ->setName($teamItem['team_affiliation']['externalid'] . ' - auto-create during import');
+                        ->setName($teamItem['team_affiliation']['externalid'] . ' - auto-create during import')
+                        ->setShortname($teamItem['team_affiliation']['externalid'] . ' - auto-create during import');
```

Comment:
This is unrelated?

Reply:
It is but it breaks the test if I don’t add this. Short name is required